internal/api/stages/fakegithub/github.go

```diff
@@ -111,7 +119,7 @@ func (f *FakeGitHub) SetCommits(r []*github.RepositoryCommit) {
 }
 
 func (f *FakeGitHub) SetReviews(r []*github.PullRequestReview) {
-	f.reviews = r
+	f.reviews = append(f.reviews, r...)
```

Comment:
why appending instead of overwriting?

Reply:
Replacing prevents us from chaining approval stage functions such as `AliceApprovesPullRequest` and `BobApprovesPullRequest`